src/main/webapp/i18n/de/featureOverview.json

```diff
@@ -80,6 +80,11 @@
                     "title": "Notenschlüssel Erstellen",
                     "shortDescription": "Erstelle einen Notenschlüssel mit benutzerdefinierten oder vorgegebenen Notenstufen für einen Kurs oder eine Prüfung.",
                     "descriptionTextOne": "Punkte sind hilfreich, um einzelne Übungen zu bewerten, aber am Ende eines Semesters möchten du und die Studierenden eine Note erhalten. Auf der Notenschlüssel-Seite kannst du für jeden Kurs und jede Prüfung einen Notenschlüssel festlegen und die Notenstufen definieren, indem du die Intervalle, die Mindestnote und die untere/obere Grenze angibst. Der Standard-Notenschlüssel kann auch mit einem einzigen Klick erstellt werden. Artemis kümmert sich dann automatisch um die Zuordnung von Noten zu Notenstufen oder Bonuspunkten und um die Berechnung der Ergebnisstatistik auf der Grundlage der Notenstufen."
+                },
+                "liveStatistics": {
+                    "title": "Live Statistik",
+                    "shortDescription": "Mithilfe der Live Statistik können Lehrende die Interaktionen der Studierenden mit der Klausur sammeln. Dies bietet weitere Möglichkeiten für Statistiken.",
```

Comment:
```suggestion
                    "title": "Live-Statistik",
                    "shortDescription": "Mithilfe der Live-Statistik können Lehrende die Interaktionen der Studierenden mit der Klausur sammeln. Dies bietet weitere Möglichkeiten für Statistiken.",
```

Reply:
Since the term is `Live Statistik`, i would not add the dash.

Reply:
I would add it everywhere then, as I think in German the two words combine into a proper compound noun either like ‘Livestatistik’ or ‘Live-Statistik’ similar to a news/sports ‘Liveticker’ or ‘Live-Ticker’ (in that case the first option is more common I think, maybe ‘Livestatistik’ is the nicer option here as well).

Reply:
Mhm, I guess we can discuss this in a future PR - for now it's fine the way it is :+1: